src/core/algorithms/gwo.cpp

```diff
@@ -0,0 +1,61 @@
+#include <stdexcept>
+#include <core/algorithms/gwo.h>
+
+namespace dnn_opt
+{
+namespace core
+{
+namespace algorithms
+{
+
+void gwo::optimize()
+{
+
+}
+
+solution* gwo::get_best()
+{
+
+}
+
+void gwo::set_params(std::vector<float> &params)
+{
+
+}
+
+void gwo::reset()
+{
+
+}
+
+void gwo::update_Elite()
+{
+  for(int i =0;i<=solution.size();i++){
```

Comment:
Code standards suggest to leave an empty space before and after aritmetic operators, also a single space after the semicolons. In addition I would suggest creating a new variable `int n = solution.size()` instead of calling the `size()` method every time. Finally, I dont see where the `solution` variable comes from?